### PR TITLE
K8SPXC-257 Pod specific serviceAccount

### PR DESCRIPTION
**Pull request by cap1984**
GKE cluster shall be created with --local-ssd-count option for
this test

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | failed |
| recreate | failed |
| big-data | passed |
| init-deploy | failed |
| scaling | failed |
| restore-to-encrypted-cluster | passed |
| scaling-proxysql | passed |
| limits | passed |
| self-healing | failed |
| upgrade | passed |
| demand-backup | passed |
| self-healing-advanced | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| security-context | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| scheduled-backup | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| big-data | failed |
| scaling | passed |
| init-deploy | failed |
| recreate | failed |
| limits | passed |
| scaling-proxysql | passed |
| monitoring | failed |
| restore-to-encrypted-cluster | passed |
| monitoring-2-0 | failed |
| affinity | passed |
| self-healing | passed |
| one-pod | failed |
| upgrade | passed |
| auto-tuning | passed |
| demand-backup | passed |
| self-healing-advanced | passed |
| proxysql-sidecar-res-limits | passed |
| upgrade-consistency | passed |
| security-context | passed |
| operator-self-healing | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| scaling | failed |
| init-deploy | passed |
| big-data | passed |
| storage | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | passed |
| upgrade | passed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| demand-backup | failed |
| security-context | failed |
| affinity | passed |
| one-pod | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| recreate | passed |
| affinity | passed |
| restore-to-encrypted-cluster | passed |
| one-pod | passed |
| demand-backup | failed |
| auto-tuning | passed |
| scaling | failed |
| proxysql-sidecar-res-limits | passed |
| big-data | passed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | failed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| storage | passed |
| recreate | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| big-data | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | failed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| storage | passed |
| self-healing | passed |
| scaling | passed |
| self-healing-advanced | passed |
| scaling-proxysql | passed |
| operator-self-healing | passed |
| upgrade | passed |
| recreate | passed |
| big-data | passed |
| upgrade-consistency | passed |
| restore-to-encrypted-cluster | passed |
| security-context | failed |
| demand-backup | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | failed |
| recreate | failed |
| big-data | failed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| restore-to-encrypted-cluster | failed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| init-deploy | passed |
| big-data | passed |
| scaling | passed |
| storage | passed |
| self-healing | passed |
| limits | passed |
| scaling-proxysql | passed |
| monitoring | passed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring-2-0 | failed |
| self-healing-advanced | passed |
| upgrade-consistency | failed |
| affinity | passed |
| security-context | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| one-pod | failed |
| auto-tuning | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| init-deploy | failed |
| scaling | failed |
| big-data | failed |
| recreate | failed |
| limits | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | failed |
| monitoring | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | failed |
| demand-backup | passed |
| scaling-proxysql | failed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| upgrade | passed |
| upgrade-consistency | failed |
| auto-tuning | passed |
| security-context | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| scaling | passed |
| big-data | passed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| restore-to-encrypted-cluster | failed |
| demand-backup | passed |
| scheduled-backup | failed |
| upgrade | passed |
| upgrade-consistency | failed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| security-context | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| big-data | passed |
| init-deploy | failed |
| scaling | passed |
| storage | passed |
| limits | passed |
| monitoring | passed |
| self-healing | failed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | failed |
| upgrade | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| demand-backup | failed |
| security-context | failed |
| one-pod | failed |
| scheduled-backup | failed |
| auto-tuning | passed |
| demand-backup-encrypted-with-tls | failed |
| proxysql-sidecar-res-limits | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | failed |
| recreate | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | failed |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| self-healing-advanced | failed |
| scheduled-backup | failed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| demand-backup-encrypted-with-tls | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| operator-self-healing | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | failed |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| self-healing-advanced | failed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | passed |
| storage | passed |
| self-healing | failed |
| scaling | passed |
| big-data | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| self-healing-advanced | failed |
| operator-self-healing | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| recreate | passed |
| proxysql-sidecar-res-limits | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| big-data | passed |
| storage | passed |
| self-healing | failed |
| self-healing-advanced | passed |
| operator-self-healing | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | passed |
| big-data | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| storage | passed |
| affinity | passed |
| self-healing | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| recreate | passed |
| scaling | passed |
| init-deploy | passed |
| big-data | failed |
| self-healing | failed |
| scaling-proxysql | passed |
| limits | passed |
| self-healing-advanced | failed |
| restore-to-encrypted-cluster | passed |
| upgrade | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| operator-self-healing | failed |
| monitoring-2-0 | passed |
| security-context | failed |
| demand-backup | passed |
| affinity | passed |
| one-pod | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| scaling | passed |
| demand-backup | passed |
| scaling-proxysql | passed |
| scheduled-backup | failed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | failed |
| big-data | failed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| self-healing-advanced | passed |
| operator-self-healing | failed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| big-data | failed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| storage | passed |
| init-deploy | passed |
| self-healing | passed |
| limits | passed |
| self-healing-advanced | passed |
| monitoring | passed |
| operator-self-healing | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | failed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | failed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| big-data | failed |
| demand-backup-encrypted-with-tls | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| scaling-proxysql | passed |
| big-data | failed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | failed |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| self-healing | passed |
| monitoring | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| limits | passed |
| upgrade | passed |
| monitoring | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | passed |
| security-context | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| storage | passed |
| big-data | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| recreate | passed |
| monitoring-2-0 | passed |
| restore-to-encrypted-cluster | passed |
| affinity | passed |
| demand-backup | passed |
| one-pod | passed |
| scheduled-backup | failed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| storage | passed |
| self-healing | passed |
| big-data | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| init-deploy | passed |
| scaling | passed |
| limits | passed |
| scaling-proxysql | passed |
| monitoring | passed |
| upgrade | passed |
| monitoring-2-0 | passed |
| upgrade-consistency | passed |
| affinity | passed |
| security-context | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| big-data | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| big-data | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| recreate | passed |
| proxysql-sidecar-res-limits | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| storage | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-417-aefc4ce0

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| storage | passed |
| recreate | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| scheduled-backup | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| big-data | passed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |

File: deploy/cr.yaml [hunk 1]
```diff
@@ -38,6 +38,7 @@ spec:
 #      runAsUser: 1001
 #      runAsGroup: 1001
 #      supplementalGroups: [1001]
+#    serviceAccountName: percona-xtradb-cluster-operator-workload
```
Comment (delgod) [inline deploy/cr.yaml:41]: Could we also add serviceAccountName into ProxySQL section?

File: pkg/apis/pxc/v1/pxc_types.go [hunk 2]
```diff
@@ -371,6 +373,10 @@ func (cr *PerconaXtraDBCluster) CheckNSetDefaults(serverVersion *ServerVersion)
 			c.PXC.LivenessInitialDelaySeconds = &livenessInitialDelaySeconds
 		}
 
+		if len(c.PXC.ServiceAccountName) == 0 {
+			c.PXC.ServiceAccountName = WorkloadSA
+		}
+
```
Comment (delgod) [inline pkg/apis/pxc/v1/pxc_types.go:379]:
```suggestion

		if len(c.ProxySQL.ServiceAccountName) == 0 {
			c.ProxySQL.ServiceAccountName = WorkloadSA
		}
```

File: deploy/cr.yaml (hunk 1 above)
Comment (cap1984) [reply deploy/cr.yaml:41]: done

File: pkg/apis/pxc/v1/pxc_types.go (hunk 2 above)
Comment (cap1984) [reply pkg/apis/pxc/v1/pxc_types.go:379]: done

Comment (cap1984) [reply pkg/apis/pxc/v1/pxc_types.go:379]: done

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| storage | failed |
| scaling | passed |
| big-data | passed |
| recreate | failed |
| self-healing | failed |
| limits | passed |
| scaling-proxysql | passed |
| self-healing-advanced | passed |
| monitoring | passed |
| upgrade | passed |
| operator-self-healing | passed |
| restore-to-encrypted-cluster | passed |
| monitoring-2-0 | passed |
| upgrade-consistency | passed |
| security-context | passed |
| affinity | passed |
| demand-backup | passed |
| one-pod | passed |
| auto-tuning | passed |
| scheduled-backup | failed |
| proxysql-sidecar-res-limits | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| storage | passed |
| big-data | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | failed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| storage | passed |
| big-data | passed |
| scheduled-backup | failed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| init-deploy | passed |
| limits | passed |
| monitoring | passed |
| monitoring-2-0 | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| init-deploy | passed |
| limits | passed |
| restore-to-encrypted-cluster | passed |
| monitoring | passed |
| demand-backup | passed |
| monitoring-2-0 | passed |
| scheduled-backup | failed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| storage | passed |
| security-context | passed |
| self-healing | passed |
| self-healing-advanced | passed |
| operator-self-healing | passed |
| big-data | passed |
| demand-backup-encrypted-with-tls | passed |

**Comment by JNKPercona** [comment]
PXC operator docker - perconalab/percona-xtradb-cluster-operator:PR-417-c3b384c2

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| init-deploy | passed |
| storage | passed |
| limits | passed |
| self-healing | passed |
| monitoring | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| recreate | passed |
| restore-to-encrypted-cluster | passed |
| demand-backup | passed |
| scheduled-backup | passed |
| scaling | passed |
| scaling-proxysql | passed |
| upgrade | passed |
| upgrade-consistency | passed |
| security-context | passed |
| demand-backup-encrypted-with-tls | passed |

File: e2e-tests/storage/compare/statefulset_hostpath-pxc.yml [hunk 3]
```diff
@@ -129,6 +131,8 @@ spec:
         imagePullPolicy: IfNotPresent
         name: pxc-init
         resources: {}
+        securityContext:
+          privileged: false
```
Comment (delgod) [inline e2e-tests/storage/compare/statefulset_hostpath-pxc.yml:135]: hm, is it really two times exists in one yaml? looks like a duplicate in 108-109 lines

Comment (cap1984) [reply e2e-tests/storage/compare/statefulset_hostpath-pxc.yml:135]: really. Those lines were taken from kubernetes

**Pull request merged**